apis/sgv2-quarkus-common/src/main/resources/application.yaml

```diff
@@ -0,0 +1,4 @@
+quarkus:
+  log:
+    console:
+      format: "%-5p [%t] %d{yyyy-MM-dd HH:mm:ss,SSS} %F:%L - %s%e%n"
```

Comment:
Two things:

* `%s%e` could this simply be `%m` - `Renders the log message plus exception (if any).`
* does `%d{ISO8601}` not working?

Reply:
Sure, changing %s%e to %m now.
ISO8601 didn't work, tested that, so had to put the date format explicitly